pyk/src/pyk/utils.py

```diff
@@ -649,11 +655,6 @@ def ensure_dir_path(path: str | Path) -> Path:
     return path
 
 
-# Implementation because of outdated Python versions: https://github.com/python/cpython/blob/1de4395f62bb140563761ef5cbdf46accef3c550/Lib/pathlib.py#L554
-def is_relative_to(_self: Path, other: Path) -> bool:
-    return _self == other or other in _self.parents
-
-
```

Comment:
This is an unrelated cleanup, but since it is straightforward to adjust for in dependents I decided to include it in this PR.